easybuild/easyconfigs/p/pasta/pasta-1.8.5-GCC-8.2.0-2.31.1-Python-3.7.2.eb

```diff
@@ -0,0 +1,60 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+easyblock = 'PythonPackage'
+
+name = 'pasta'
+version = '1.8.5'
+versionsuffix = '-Python-%(pyver)s'
+local_commit = '926a4eb'
+
+homepage = 'https://github.com/smirarab/pasta'
+description = """PASTA (Practical Alignment using SATe and Transitivity)"""
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
+
+github_account = 'smirarab'
+source_urls = [
+    GITHUB_SOURCE,
+    'https://github.com/smirarab/sate-tools-linux/archive'
+]
+sources = [
+    {
+     'download_filename': '%s.tar.gz' % local_commit,
```

Comment:
@omula You'll need two levels of indentation (4*2=8 spaces) here to make the style check pass, same below